webapp/components/User/User.vue

```diff
@@ -79,7 +79,7 @@
             />
           </ds-flex-item>
           <ds-flex-item :width="{ base: 1 }">
-            <ds-button fullwidth>
+            <ds-button fullwidth @click="block(user)">
```

Comment:
Honestly, I would like to get this button completely removed. It's enough to have it on the user profile page.

This button exists since the early days of our nitro code base. It never worked and this has been discovered just a few days ago. Apparently not too many users expect the block button in the popup.

Reply:
Oh, didn't know that! I'm fine with removing it – @ogerly what do you think?

Reply:
dennis has asked me to do this. Please discuss this with dennis.